indexers/keyvalue/MongoDBIndexer/__init__.py

```diff
@@ -83,14 +83,16 @@ def query_handler(self) -> 'MongoDBHandler':
         """Get the handler to MongoDB."""
         return self.get_query_handler()
 
-    def query(self, key: str, *args, **kwargs) -> Optional[bytes]:
+    def query(self, keys: Iterable[str], *args, **kwargs) -> Optional[bytes]:
         """Query the serialized documents by document id.
 
         :param key: document id
         :return: serialized document
         """
+        result = []
         with self.query_handler as mongo_handler:
-            result = mongo_handler.query(key)
+            for key in keys:
```

Comment:
would be good to see if the mongo handler can handle itself multiple queries at the same time

Reply:
tried but did not work

Reply:
see if can be changed to make it work

Reply:
```
result = mongo_handler.query(keys)
```

Reply:
change [mongodbhandler.py:L79](https://github.com/jina-ai/jina-hub/blob/master/indexers/keyvalue/MongoDBIndexer/mongodbhandler.py#L79) to 
```
cursor = self.collection.find({'_id': {"$in": keys}})
```

reference:

https://docs.mongodb.com/manual/tutorial/query-documents/#specify-conditions-using-query-operators

Reply:
I think this should work.

Reply:
The problem may come from the fact that you cannot keep track of the `failing` keys. 

So maybe it is better to keep the single batch inside the `mongohandler`

Reply:
This change that nan proposes works, the issue is that then I need to iterate over the returned list and verify the ids to check for missing ids;

I can do 
```aux = mongo_handler.query(['fccf9b08-acaf-11eb-bd09-0242ac110002', '221-fakeid-221',  'fccfda14-acaf-11eb-bd09-0242ac110002'])```

But then `len(aux)` is 2, not 3, and I need a loop over items.